cert/lib/cert/runner.rb

```diff
@@ -223,17 +227,21 @@ def create_certificate
 
       cert_path = store_certificate(certificate, Cert.config[:filename])
 
-      # Import all the things into the Keychain
-      keychain = File.expand_path(Cert.config[:keychain_path])
-      password = Cert.config[:keychain_password]
-      FastlaneCore::KeychainImporter.import_file(private_key_path, keychain, keychain_password: password, skip_set_partition_list: Cert.config[:skip_set_partition_list])
-      FastlaneCore::KeychainImporter.import_file(cert_path, keychain, keychain_password: password, skip_set_partition_list: Cert.config[:skip_set_partition_list])
+      if Helper.mac?
+        # Import all the things into the Keychain
+        keychain = File.expand_path(Cert.config[:keychain_path])
+        password = Cert.config[:keychain_password]
+        FastlaneCore::KeychainImporter.import_file(private_key_path, keychain, keychain_password: password, skip_set_partition_list: Cert.config[:skip_set_partition_list])
+        FastlaneCore::KeychainImporter.import_file(cert_path, keychain, keychain_password: password, skip_set_partition_list: Cert.config[:skip_set_partition_list])
+      else
+        UI.message("Skipping importing certificates as it would not work on this operating system.")
+      end
```

Comment:
[self-review] Importing is optional, main purpose is to create .p12 and .cer files and commit to remote.